bundle/manifests/codeflare-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,549 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "codeflare.codeflare.dev/v1alpha1",
+          "kind": "InstaScale",
+          "metadata": {
+            "labels": {
+              "app.kubernetes.io/created-by": "codeflare-operator",
+              "app.kubernetes.io/instance": "instascale-sample",
+              "app.kubernetes.io/managed-by": "kustomize",
+              "app.kubernetes.io/name": "instascale",
+              "app.kubernetes.io/part-of": "codeflare-operator"
+            },
+            "name": "instascale-sample"
+          },
+          "spec": {}
+        },
+        {
+          "apiVersion": "codeflare.codeflare.dev/v1alpha1",
+          "kind": "MCAD",
+          "metadata": {
+            "name": "mcad-sample"
+          },
+          "spec": {}
+        }
+      ]
+    capabilities: Basic Install
+    operators.operatorframework.io/builder: operator-sdk-v1.24.1
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+    operatorframework.io/suggested-namespace: openshift-operators
+  name: codeflare-operator.v0.0.1
+  namespace: placeholder
```

Comment:
Perhaps add the repository and containerImage to the metadata. For example:

```
      repository: https://github.com/project-codeflare/codeflare-operator.git
      containerImage: <img registry >/<namspace>/<repo name>:v0.0.1
```

Reply:
I'm not seeing it under the csv spec. Are you saying additional links, like so? The below code snippet is basically from the bottom of the file

```
  links:
  - name: Codeflare Operator
    url: https://codeflare.dev/
  - name: Container image
    url: quay.io/project-codeflare/codeflare-operator:0.0.1
  - repository: https://github.com/project-codeflare/codeflare-operator.git
  maintainers:
  - email: aasthana@redhat.com
    name: Anish Asthana
  - email: kpostlet@redhat.com
    name: Kevin Postlethwait
  - email: meyceoz@redhat.com
    name: Mustafa Eyceoz
  maturity: alpha
  provider:
    name: CodeFlare
    url: https://github.com/project-codeflare
  version: 0.0.1
```

I"m not necessarily seeing the value of additional links as:
1) The container Image is already specified in the CSV deployment spec
2) The repository is linked under "provider"

Reply:
Ted explained to me offline what he meant, I've made the updates.